test/factories/puppet_related.rb

```diff
@@ -5,7 +5,7 @@
 
   factory :environment_class
 
-  factory :lookup_key do
+  factory :puppetclass_lookup_key do
```

Comment:
You could use inheritance in factory so you don't DRY yourself
(http://stackoverflow.com/a/20101117/973901)